it/config/DocumentationControllerISpec.scala

```diff
@@ -67,6 +66,8 @@ class DocumentationControllerISpec extends IntegrationBaseSpec {
        |}
     """.stripMargin)
 
+  private val headerRegex = """(?s).*?application/vnd\.hmrc\.(\d+\.\d+)\+json.*?""".r
```

Comment:
Can you inline and move this headerRegex closer to the test as its only used there, makes it more readable.